Documentation/FrameworkOverview.md

```diff
@@ -86,56 +147,7 @@ Some subjects offer additional behaviors as well. In particular,
 [subscribers](#subscription), like when a network request finishes before
 anything is ready to handle the result.
 
-### Commands
-
-A **command**, represented by the [RACCommand][] class, creates and subscribes
-to a signal in response to some action. This makes it easy to perform
-side-effecting work as the user interacts with the app.
-
-Usually the action triggering a command is UI-driven, like when a button is
-clicked. Commands can also be automatically disabled based on a signal, and this
-disabled state can be represented in a UI by disabling any controls associated
-with the command.
-
-On OS X, RAC adds a `rac_command` property to
-[NSButton][NSButton+RACCommandSupport] for setting up these behaviors
-automatically.
-
-### Connections
-
-A **connection**, represented by the [RACMulticastConnection][] class, is
-a [subscription](#subscription) that is shared between any number of
-subscribers.
-
-[Signals](#signals) are _cold_ by default, meaning that they start doing work
-_each_ time a new subscription is added. This behavior is usually desirable,
-because it means that data will be freshly recalculated for each subscriber, but
-it can be problematic if the signal has side effects or the work is expensive
-(for example, sending a network request).
-
-A connection is created through the `-publish` or `-multicast:` methods on
-[RACSignal][RACSignal+Operations], and ensures that only one underlying
-subscription is created, no matter how many times the connection is subscribed
-to. Once connected, the connection's signal is said to be _hot_, and the
-underlying subscription will remain active until _all_ subscriptions to the
-connection are [disposed](#disposables).
-
-## Sequences
-
-A **sequence**, represented by the [RACSequence][] class, is a _pull-driven_
-[stream](#streams).
-
-Sequences are a kind of collection, similar in purpose to `NSArray`. Unlike
-an array, the values in a sequence are evaluated _lazily_ (i.e., only when they
-are needed) by default, potentially improving performance if only part of
-a sequence is used. Just like Cocoa collections, sequences cannot contain `nil`.
-
-Sequences are similar to [Clojure's sequences][seq] ([lazy-seq][] in particular), or
-the [List][] type in [Haskell][].
-
-RAC adds a `-rac_sequence` method to most of Cocoa's collection classes,
-allowing them to be used as [RACSequences][RACSequence] instead.
-
+<!-- TODO: Update -->
 ## Disposables
```

Comment:
Should tie in with `Interrupted`, but again, feel free to leave that as a placeholder if you want.